config/routes.rb

```diff
@@ -32,9 +32,10 @@
     resources :allocations, only: %i[show update edit], as: :user_allocations
     resources :users, only: %i[show update edit], as: :admin_user
     resources :punches, only: :show, as: :user_punches
-    resources :regional_holidays, except: %i[show update]
+    resources :regional_holidays, except: %i[show update destroy]
     resources :regional_holidays, only: :show, as: :show_regional_holiday
     resources :regional_holidays, only: :update, as: :update_regional_holiday
+    resources :regional_holidays, only: :destroy, as: :destroy_regional_holiday
```

Comment:
now that I noticed these duplicated declarations of routes, do you know why?

Reply:
I intended to name specific resources for `regional_holidays` in order to not conflict with ActiveAdmin's regional_holidays routes. In this case, `show`, `update` and `destroy` were declared separately to achieve this goal.